RealmSwift-swift1.2/Tests/TestCase.swift

```diff
@@ -23,6 +23,18 @@ import Realm.Dynamic
 import RealmSwift
 import XCTest
 
+func XCTAssertEqual<T: Any where T: Equatable>(@autoclosure a: () -> T?, @autoclosure b: () -> T?, _ message: String? = nil, file: String = __FILE__, line: UInt = __LINE__) {
+    if let _a = a() {
+        if let _b = b() {
+            XCTAssertEqual(_a, _b, (message != nil ? message! : ""), file: file, line: line)
+        } else {
+            XCTFail((message != nil ? message! : "a != nil, b == nil"), file: file, line: line)
+        }
+    } else if let _b = b() {
```

Comment:
don't you get a compiler warning here for not using `_b`? Shouldn't this be `_ = b()` instead?